espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Handlers/Exceptions/AppiumException.java

```diff
@@ -0,0 +1,12 @@
+package io.appium.espressoserver.lib.Handlers.Exceptions;
+
+public abstract class AppiumException extends Exception {
```

Comment:
Not really. I just figured it's not going to ever be used directly so declare it abstract. I could make it an interface, I suppose.